containers/ecr-viewer/seed-scripts/create-seed-data.py

```diff
@@ -1,156 +1,103 @@
 import argparse
-import json
+import io
 import os
+import zipfile
 
 import grequests
 
-URL = "http://orchestration-service:8080"
+UPLOAD_URL = "http://host.docker.internal:3000/ecr-viewer/api/process-zip"
 BASEDIR = os.path.dirname(os.path.abspath(__file__))
 
 
 def _get_args():
     parser = argparse.ArgumentParser(
-        prog="Create Seed Data",
-        description="Convert eICR and RR files to FHIR bundles and insert them into the database.",
-        epilog="For each directory in baseECR the script will look for a `CDA_eICR.xml` and `CDA_RR.xml` file. If they are found, it will convert them into a FHIR bundle (saved as `bundle.json`) and insert that into the database using the Orchestration service.",
-    )
-    parser.add_argument(
```

Comment:
is there any way for `process-zip` to skip convert? and/or does it always skip convert if there's already a bundle?

Reply:
@JNygaard-Skylight will that cause any issues? I think you were the one who added the skipping